src/transformers/image_processing_base.py

```diff
@@ -0,0 +1,542 @@
+# coding=utf-8
```

Comment:
This is just moving objects from image_processing_utils.py to this new file, which will share the common objects between the base fast and slow image processors in image_processing_utils.py and image_processing_utils_fast.py.

This is to match the structure of the tokenizers files